src/app/vm/vm-sidebar/affinity-group-selector/affinity-group-list/affinity-group-list.component.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges
+} from '@angular/core';
+import { AffinityGroup, AffinityGroupTypesMap } from '../../../../shared/models';
+
+
+@Component({
+  selector: 'cs-affinity-group-list',
+  templateUrl: 'affinity-group-list.component.html',
+  styleUrls: ['affinity-group-list.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class AffinityGroupListComponent implements OnChanges {
+  @Input() public affinityGroups: AffinityGroup[];
+  @Input() public isVmCreation: boolean;
```

Comment:
Can't we create several components instead of using `isVmCreation: boolean` to modify behavior?

For example:
VmCreationAffinityGroupListComponent
SomeOtherAffinityGroupListComponent
AffinityGroupListComponent